README.md

```
@@ -100,61 +119,31 @@ This repository also contains scripts that were used to *generate* the configura
 *   For detailed information on how to use these scripts, their structure, and contribution guidelines for them, please refer to `scripts/README.md`.
 *   **Caution:** Modifying and running these generator scripts will overwrite the configuration files within the repository itself.
 
-## 7. Troubleshooting
+## Troubleshooting
 
 *   **Fonts not displaying correctly (especially icons):**
-    *   Ensure JetBrainsMono Nerd Font and Font Awesome are installed and recognized.
-    *   You might need to rebuild your font cache: `fc-cache -fv`.
+  *   Ensure JetBrainsMono Nerd Font and Font Awesome are installed and recognized.
```

markdownlint-cli2 (0.17.2)

Inconsistent indentation for list items at the same level
Expected: 0; Actual: 2

(MD005, list-indent)